src/current/_includes/releases/v24.2/v24.2.4.md

```diff
@@ -0,0 +1,95 @@
+
+<h3 id="v24-2-4-bug-fixes">Bug fixes</h3>
+
+- Addressed a bug in the [upgrade]({% link v24.2/upgrade-cockroach-version.md %}) pre-condition for repairing descriptor corruption, which could lead to upgrade finalization being stuck. [#130517][#130517]
+- Fixed a potential memory leak in [changefeeds]({% link v24.2/change-data-capture-overview.md %}) using a [cloud storage sink]({% link v24.2/changefeed-sinks.md %}#cloud-storage-sink). The memory leak could occur if both [`changefeed.fast_gzip.enabled`]({% link v24.2/cluster-settings.md %}#setting-changefeed-fast-gzip-enabled) and `changefeed.cloudstorage.async_flush.enabled` were `true`, and the changefeed received an error while attempting to write to the cloud storage sink. [#130602][#130602]
+- Fixed a bug where zone configuration changes issued by the [declarative schema changer]({% link v24.2/online-schema-changes.md %}#declarative-schema-changer) were not blocked if a table had `schema_locked` set. [#130705][#130705]
+- Fixed a bug in which some [`SELECT FOR UPDATE`]({% link v24.2/select-for-update.md %}) or [`SELECT FOR SHARE`]({% link v24.2/select-for-update.md %}) queries using `NOWAIT` could still block on locked rows when using the `optimizer_use_lock_op_for_serializable` [session setting]({% link v24.2/session-variables.md %}) under [`SERIALIZABLE`]({% link v24.2/demo-serializable.md %}) isolation. This bug was introduced with [`optimizer_use_lock_op_for_serializable`]({% link v24.2/session-variables.md %}#optimizer-use-lock-op-for-serializable) in v23.2.0. [#130430][#130430]
+- Fixed a bug that caused the [optimizer]({% link v24.2/cost-based-optimizer.md %}) to plan unnecessary post-query uniqueness checks during [`INSERT`]({% link v24.2/insert.md %}), [`UPSERT`]({% link v24.2/upsert.md %}), and [`UPDATE`]({% link v24.2/update.md %}) statements on tables with [partial]({% link v24.2/partial-indexes.md %}), [unique]({% link v24.2/create-index.md %}#unique-indexes) [hash-sharded indexes]({% link v24.2/hash-sharded-indexes.md %}). These unnecessary checks added overhead to execution of these statements, and caused the statements to error when executed under [`READ COMMITTED`]({% link v24.2/read-committed.md %}) isolation. [#130570][#130570]
+- Fixed a bug that could result in the inability to garbage collect an [MVCC]({% link v24.2/architecture/storage-layer.md %}#mvcc) range tombstone within a [global table]({% link v24.2/global-tables.md %}). [#130940][#130940]
+- Previously, if a connection was attempting a [schema change]({% link v24.2/online-schema-changes.md %}) while the same schema objects were being dropped, it was possible for the connection to be incorrectly dropped. This is now fixed. [#130962][#130962]
```

Comment:
```suggestion
- Fixed a bug where a connection could be dropped if the client was attempting a [schema change]({% link v24.2/online-schema-changes.md %}) while the same schema objects were being dropped, it was possible for the connection to be incorrectly dropped. [#130962][#130962]
```

Reply:
```suggestion
- Fixed a bug where a connection could be dropped if the client was attempting a [schema change]({% link v24.2/online-schema-changes.md %}) while the same schema objects were being dropped. [#130962][#130962]
```

I think this was your intended edit. LGTM